sample-code/src/tutorials/mcp/weather-mcp-server.ts

```diff
@@ -0,0 +1,58 @@
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { z } from "zod";
+
+const server = new McpServer({
+  name: "Weather Server",
+  version: "1.0.0"
+});
+
+server.tool(
```

Comment:
[pp] maybe point out that typically this part is not necessary, as one will often use provided / remote MCP servers, and only included here to have an independent example